.travis.yml

```diff
@@ -5,8 +5,18 @@ matrix:
   fast_finish: true
   include:
     - if: repo = arvidn/libtorrent
-      env: variant=release sonar_scan=1 toolset=gcc pylint=1
-    - env: variant=test_debug lint=1 tests=1 toolset=gcc-sanitizer
+      env: variant=release sonar_scan=1 toolset=gcc
+    - env: variant=debug toolset=gcc lint=1 pylint=1 clang_tidy=1
+      addons:
+        apt:
+          sources:
+          - ubuntu-toolchain-r-test
+          packages:
+          - libc++-dev
+          - python3-pip
+          - libboost1.55-all-dev
```

Comment:
question: are these packages really needed here? (they are already included below)

Reply:
I originally thought these packages were added to the ones mentioned below, but after trying, it appears they *replace* the packages below.